joomla/joomla-cms
# [4.1] Form list with showon attributes should load the show on script

Pull Request for Issue #37299

### Summary of Changes

- Load the showon script if there are showon attributes even in the options (one level down)

### Testing Instructions

- Open file: `\joomla-cms\administrator\components\com_banners\forms\banner.xml`
- Find the state field and replace with the following

```xml
		<field
			name="state"
			type="list"
			label="JSTATUS"
			class="form-select-color-state"
			default="1"
			validate="options"
			showon="state!:X"
			>
			<option value="1" showon="sticky:0">JPUBLISHED</option>
			<option value="0" showon="sticky:0">JUNPUBLISHED</option>
			<option value="2" showon="sticky:0">JARCHIVED</option>
			<option value="-2" showon="sticky:0">JTRASHED</option>
		</field>
```
- Create a new banner: `/administrator/index.php?option=com_banners&view=banner&layout=edit`
- Toggle Banner's Pinned button. You'll see the Status options disappear after toggling. (You'll first have to click the status list in order to see it change. It would be better for it to update, but that's another issue)
- Now remove the `showon="state!:X"` line from the code above. Press CTRL+F5.
- Retry toggling, it's broken now.

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required
